README.md

```diff
@@ -1,20 +1,60 @@
-# Tripper
+Tripper
+=======
+_Triplestore wrapper for Python providing a simple and consistent interface to a range of triplestore backends._
 
-**A triplestore wrapper for Python.**
+[![PyPI](https://img.shields.io/pypi/v/tripper?logo=pypi)](https://pypi.org/project/tripper)
+[![Documentation](https://img.shields.io/badge/documentation-informational?logo=github)](https://emmc-asbl.github.io/tripper/latest/)
+![CI tests](https://github.com/EMMC-ASBL/tripper/workflows/CI%20-%20Tests/badge.svg)
 
-_The best ride when handling any triplestore._
 
-See the [documentation](docs/overview.md) for a quick tutorial.
+Basic concepts
+--------------
+**Tripper** strives for simplicity and is modelled after [rdflib] (with a few simplifications).
 
+In Tripper are:
+* All IRIs are represented by Python strings.
+  Example: `"http://emmo.info/emmo#Atom"`
 
-## Installation
+* Blank nodes are strings starting with "_:".
+  Example: `"_:bnode1"`
 
-The package can be installed from [PyPI](https://pypi.org/project/tripper) using `pip`:
+* Literals are constructed with [`tripper.Literal`][Literal].
+  Example: `tripper.Literal(3.14, datatype=XSD.float)`
+
+To make it easy to work with IRIs, provide Tripper a set of pre-defined namespaces, like `XSD.float`.
+New namespaces can easily be defined with the [`tripper.Namespace`][Namespace] class.
+
+A triplestore wrapper is created with the [`tripper.Triplestore`][Triplestore] class.
+
+
+
+Documentation
+-------------
+* Getting started: Take a look at the [tutorial](docs/tutorial.md).
+* Reference manual: [API Reference]
+
+
+Installation
+------------
+**Tripper** has by itself no dependencies outside the standard
+library, but the triplestore backends may have.
```

Comment:
```suggestion
Tripper has by itself no dependencies outside the standard library, but the triplestore backends may have specific dependencies.
```

Reply:
`git` is terrible at handling standard text. For ease of reading `git diff`s and reviewing documentation, keeping the philosophy of 1 sentence per 1 line, provides more easily readable `git diff`s for most cases. There might still be some things that are difficult to parse as a human, when reviewing, but it drastically improves readability.

I much prefer reviewer readability support than terminal viewer support of the raw markdown. If one want to read the documentation locally, I suggest to run the MkDown server locally (`mkdocs serve`) instead and browse the documentation in a web browser from there.